common4j/src/main/com/microsoft/identity/common/java/crypto/StorageEncryptionManager.java

```diff
@@ -189,44 +193,43 @@ public byte[] encrypt(final byte[] plaintext)
 
     @Override
     public byte[] decrypt(final byte[] cipherText) throws ClientException {
-        final String methodName = ":decrypt";
+        final String methodTag = TAG + ":decrypt";
 
         final byte[] dataBytes;
         try {
             dataBytes = stripEncodeVersionFromCipherText(cipherText);
         } catch (final ClientException e){
-            Logger.verbose(TAG + methodName,
+            Logger.verbose(methodTag,
                     "Failed to strip encode version from cipherText, string might not be encrypted. Exception: ", e.getMessage());
             return cipherText;
         }
 
         final List<AbstractSecretKeyLoader> keysForDecryption = getKeyLoaderForDecryption(cipherText);
         if (keysForDecryption.size() == 0) {
-            throw new IllegalStateException("KeyLoader list must not be null or empty.");
+            // Developer error. Throw.
```

Comment:
How is this surfaced back to the developer? I see IllegalStateException being thrown but I believe this is runtime...are we converting to something else down the line?

Reply:
In broker, yes. In MSAL, no.

The idea here is that this error should be noticeable by us before it's even released to prod, so we want to make it very loud here.

(unlike other error that could be because bugs in OS. this is 100% dev error. either we encrypt data with a wrong key, or we didn't properly load decryption key.)